TESTS/mbed_hal/ticker_api_tests/main.cpp

```diff
@@ -0,0 +1,520 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2017 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "mbed.h"
+#include "greentea-client/test_env.h"
+#include "unity.h"
+#include "utest.h"
+#include "rtos.h"
+#include "hal/us_ticker_api.h"
+#include "core_cm.h"
+
+#define FORCE_OVERFLOW_TEST (false)
+#define TICKER_INT_VAL 5000
+#define TICKER_DELTA 50
+
+#define TICKER_OVERFLOW_DELTA 50
+#if DEVICE_LOWPOWERTIMER
+#define TICKER_OVERFLOW_DELTA 0 // this will allow to detect that ticker counter rollsovers to 0
+#endif
+#define TICKER_EXAMPLE_VALUE 100
```

Comment:
This name does not say much about the use of this value. Rename or add a comment.

Reply:
Modified variable name.